evalml/tests/automl_tests/test_automl_dask.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+
+import numpy as np
+import pytest
+from distributed import Client
+
+from evalml.automl import AutoMLSearch
+from evalml.automl.callbacks import raise_error_callback
+from evalml.automl.engine import DaskEngine, SequentialEngine
+from evalml.tests.automl_tests.dask_testing import (
+    TestPipelineFast,
+    TestPipelineSlow,
+    TestPipelineWithFitError,
+    TestPipelineWithScoreError
+)
+
+
+@pytest.mark.usefixtures("X_y_binary_cls")
+class TestAutoMLSearchDask(unittest.TestCase):
+
+    @pytest.fixture(autouse=True)
+    def inject_fixtures(self, caplog):
+        """ Gives the unittests access to the logger"""
+        self._caplog = caplog
```

Comment:
Why does the caplog need to be an instance property? Won't that mean that the caplog is shared across tests? I guess I'm worried what would happen if we have two tests running in parallel, one checks that "error happened" is in caplog and the other checks that "error happened" is not in the caplog.